packages/entity-list/src/modules/list/sagas.spec.js

```diff
@@ -114,6 +114,20 @@ describe('entity-list', () => {
           })
         })
 
+        describe('setParent saga', () => {
+          test('should set parent, reload entity model, form definition and data', () => {
+            return expectSaga(sagas.setParent)
+              .provide([
+                [select(sagas.entityListSelector), {formName: 'UserTest', entityName: 'Docs_list_item'}],
+                [select(sagas.listSelector), generateState({}, 1)],
+                [matchers.call.fn(sagas.loadEntityModel), {}],
+                [matchers.call.fn(sagas.loadFormDefinition), {}],
+                [matchers.call.fn(sagas.reloadData), {}]
+              ])
+              .run()
```

Comment:
actually this test doesn't verify anything :)

the providers in `provide` are only mocks, this test would also pass if the `setParent` function was completely empty, right?

actual assertions should be added after the `provide()` before `run()` is called